cmsplugin_filer_file/templates/cmsplugin_filer_file/file.html

```diff
@@ -1,7 +1,9 @@
 {% if object.file.url %}
 <span class="file" {% if inherited_from_parent %} style="{{inherited_from_parent.style}}" {% endif %} >
 <a href="{{ object.file.url }}"{% if object.target_blank %} target="_blank"{% endif %}>
-{% if object.get_icon_url %}<img src="{{STATIC_URL}}{{ object.get_adjusted_icon_url }}" style="border: 0px" alt="Icon" />{% endif %}
+{% if object.get_icon_url %}
+    <img src="{% if object.is_absolute_url == False %}{{ PROXY_PREFIX }}{% endif %}{{ object.get_icon_url }}" style="border: 0px" alt="Icon" />
+{% endif %}
```

Comment:
I don't like having so much logic in the template. I bet we could use the `static` template tag to do this. https://docs.djangoproject.com/en/1.8/ref/templates/builtins/#static

In particular, I bet we could use the S3Storage app for our static file storage, not just for filer: http://django-storages.readthedocs.org/en/latest/backends/amazon-S3.html